lambda/query-activity-log/decrypt-data.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  EncryptionContext,
+  KmsKeyringNode,
+  buildDecrypt,
+} from "@aws-crypto/client-node";
+import buildKmsKeyring from "./kms-keyring-builder";
+
+const MAX_ENCRYPTED_DATA_KEY = 5;
+const DECODING = "utf8";
+
+const decryptClientConfig = { maxEncryptedDataKeys: MAX_ENCRYPTED_DATA_KEY };
+const decryptClient = buildDecrypt(decryptClientConfig);
+
+let keyring: KmsKeyringNode;
+
+export function generateExpectedContext(userId: string): EncryptionContext {
```

Comment:
so the accessCheckValue is not being checked?
VC store use this as an additional confirmation.  I'm not sure how much more security it really provides. 
If we're not going to use it to validate then we shouldn't write it

Reply:
I'm going to add that check in a future PR, but I wanted to keep this one as the MVP for decryption